src/main/webapp/WEB-INF/app.conf

```diff
@@ -36,7 +36,7 @@ ssource.endpoint     = http://localhost:3030/landregistry_to/query
 ssource.connectTimeout = 10000
 ssource.remoteTimeout = 3600000
 
-# API configuration - largely redundant now
+# API configuration - redundant now
```

Comment:
Then why not delete it?

Reply:
Turns out I was wrong and it is needed. I've update the comment.